pandas/_version.py

```diff
@@ -1,20 +1,25 @@
-# pylint: disable=consider-using-f-string
```

Comment:
Were changes to this file made by versioneer? I'm surprised non vendored mode didn't get rid of it. 
(I guess I'll just nuke it after we drop setuptools)

Reply:
Yes, this is from versioneer.